portainer-compose.yml

```diff
@@ -13,3 +13,5 @@ services:
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
     network_mode: "host"
+    ports:
```

Comment:
This is actually a good idea for various testing scenarios, but it should perhaps be triggered from an environment variable or other toggle rather than being "always on".